src/views/newConnection/base.tsx

```diff
@@ -69,9 +73,11 @@ export abstract class NewConnectionTemplates {
   protected feedback = (props: { feedback: FormFeedback }): JSX.Element => {
     switch (props.feedback.type) {
       case 'message':
-        return <this.feedbackMessage message={props.feedback.message} isError={false} />
+        return <this.feedbackMessage message={props.feedback.message} />
       case 'companyFound':
         return <this.feedbackCompanyInfo company={props.feedback.company} />
+      case 'success':
+        return <></>
```

Comment:
empty?

Reply:
since we agreed on feedbacks, so it won;t be for long